### ISSUE TITLE
Not receiving notifications on Android

### ISSUE THREAD

**Issue by gullitmiranda** [gitterHQ/gitter#894]
How do I enable notifications in the Android app?

In previous versions of the app that was possible.

**Comment by MadLittleMods** [comment]
@gullitmiranda Notifications should be enabled by default.

If you go to `Settings -> Sound & notification -> App notifications -> Gitter`, you can check whether you have blocked them, etc.

**Comment by gullitmiranda** [comment]
It is not blocked.

Notifications room would be interesting, but be notified when a user sends a message would be very good, because in https://github.com/azukiapp/, we use the gitter as the main form of contact for support https://github.com/azukiapp/azk/ project, since which is a tool for developers.

**Comment by MadLittleMods** [comment]
@gullitmiranda Which version of the app do you have? `Settings -> Apps -> All -> Gitter`

@trevorah Could this be something going wrong with the notifications IDs (Related to https://github.com/gitterHQ/gitter/issues/868)?

**Comment by gullitmiranda** [comment]
I use the `v2.0.1`.

Now it only mentions quotes or any message into a room I've ever interacted (or created)?

The app for Android is not open source?

**Comment by MadLittleMods** [comment]
@gullitmiranda Ahh, I thought you weren't getting any notifications. This might just be the new functionality but could be a bug. It sounds like the app is using the `mention` setting vs `all` in `notifications`.

**Comment by gullitmiranda** [comment]
I am not getting any notification :cry:
The question was just a curiosity.

**Comment by trevorah** [comment]
@gullitmiranda do you have the desktop or web app open when you are doing this?

**Comment by gullitmiranda** [comment]
I usually just open the desktop app when I receive notification in android.

**Comment by gullitmiranda** [comment]
started working again.

I will close the issue, and thank you for quick support.

**Comment by ppvg** [comment]
I don't receive **any** notifications. I'm on Gitter 2.10.0, Android 6.0.1.

What I've checked and tried:
- Gitter default setting is `all`, room settings are either `all` or `mention`.
- App notifications are enabled (default settings).
- Getting mentioned doesn't result in a notification, even with all apps closed except the android app.
- Clearing data and reinstalling the app has no effect, but I noticed I'm still signed in afterwards. Not sure why.

I **do** receive **email** notifications.
This issue would be mitigated slightly if the <kbd>Open chart</kbd> button would open in the app.
